### ISSUE TITLE
[DA_Update] Update to OpenStudio 3.0 and ruby 2.5

### ISSUE THREAD

**Issue by TMaile** [BuildingSync/BuildingSync-gem#63]
- [x] created new branch DA_update based on DA branch
- [x] changed to model-articulation-gem to v 0.2.0
- [x] changed model-articulation-gem from DA branch to develop branch
- [x] updated to support Ruby 2.5.8
- [x] installed OpenStudio 3.0.1
- [x] changed reference from ruby 2.5.8 to OpenStudio 3.0.1
- [x] refactor code that uses Epw class
- [x] adjusted travis testing to use 3.0.1 (depends on #65)
- [x] update reference files and measures to reflect changes in OpenStudio 3.0.1 (adding these measures to openstudio-ee-gem here: https://github.com/NREL/openstudio-ee-gem/commit/4d595fc49b19feff958031477952538ccfb5d912)
  - tenant_star_internal_loads
  - replace_simple_glazing
  - vr_fwith_doas
  - add_apszhp_to_each_zone
  - improve_simple_glazing_by_percentage
  - set_boiler_thermal_efficiency
  - replace_hvac_with_gshp_and_doas
  - ReplaceFanTotalEfficiency
  - ImproveFanTotalEfficiencybyPercentage
  - add_energy_recovery_ventilator
  - set_water_heater_efficiency_heat_lossand_peak_water_flow_rate
  - reduce_water_use_by_percentage
  - add_apszhp_to_each_zone
- [x] refactor code to ensure tests are passing again

**Comment by corymosiman12** [comment]
@TMaile the [compatibility matrix](https://github.com/NREL/OpenStudio/wiki/OpenStudio-SDK-Version-Compatibility-Matrix) needs ruby 2.5.5 just fyi

**Comment by TMaile** [comment]
main update is complete, created a new issue for measure problem #74
